server/backend/users/tests/test_views.py

```
@@ -0,0 +1,92 @@
+
+    def test_logout_view(self):
+        user = User.objects.create_user(username='testuser', password='password123', email='testuser@example.com', role='student')
+        
+        # Generate refresh token
+        refresh_token = str(RefreshToken.for_user(user))
+        response = self.client.post('/logout/', {'refresh': refresh_token}, format='json')
+        
+        assert response.status_code == status.HTTP_205_RESET_CONTENT
+        assert response.data.get("message") == "Logout successful"
+
+    def test_logout_view_no_token(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_logout_view_no_token`

Add return type annotation: `None`

(ANN201)